lib/testscript_engine/cli.rb

```diff
@@ -6,11 +6,12 @@ class TestScriptEngine
 
   module CLI 
     class MyCLI < Thor
-      desc "execute [OPTIONS]", "--config[PATH] --ext_validator [URL] --ext_fhirpath [URL] --variable ['name = value'] --server_url [URL] --testscript_path [FILEPATH] --testreport_path [FILEPATH] --nonfhir_fixture --verbose"
+      desc "execute [OPTIONS]", "--config[PATH] --ext_validator [URL] --ext_fhirpath [URL] --variable ['name = value'] --server_url [URL] --testscript_path [FILEPATH] --testreport_path [FILEPATH] summary_path [FILEPATH] --nonfhir_fixture --verbose"
```

Comment:
Minor thing: [FILEPATH] [PATH] [FILE] been used mixed way and should be clear. --config needs a specific file name whereas --summary_path or --testreport need a path. My vote is to use [FILE] for file and use [PATH] for path. What do you think?

Reply:
I'm onboard with [PATH]. I'll change just the summary_path stuff with this pull request and leave the others for you to change as a part of other changes you're making to the path handling. Or do you want me to roll it all into this PR?

Reply:
fixed all path stuff